README.md

```diff
@@ -199,3 +174,36 @@ stdenv.mkDerivation {
 
 </p>
 </details>
+
+<details><summary>4. nix store 占用磁盘空间过大时，如何回收空间? </summary>
+<p>
+
+首先先删除不再需要的 [gc root](https://nixos.org/manual/nix/stable/#ssec-gc-roots)。例如，如果只需要保留最近5次构建的 NixOS 版本：
+
+```sh
+sudo nix-env --profile /nix/var/nix/profiles/system --delete-generations +5
+```
+
+然后再执行 gc 操作
+
+```sh
+nix-collect-garbage
+```
```

Comment:
How about stating more on `nix-collect-garbage --delete-older-than` and `nix.gc`

Reply:
> How about stating more on `nix-collect-garbage --delete-older-than` and `nix.gc`

Can you edit this in another pr? I'll merge this if everything else looks good.